$NetBSD: patch-bv,v 1.3 2011/10/06 02:19:06 manu Exp $

Use linkat(2) if available so that we can make a hardlink to symlink
itself, instead of symlink destination, like link(2) does. This is
required to support moving symlinks on distributed volumes, as
glusterfsd really wants to hardlink the symlink and not its target.

If linkat(2) is inavailable, then moving a symlink on distributed
volumes fails.

--- config.h.in.orig	2011-08-23 14:31:46.000000000 +0200
+++ config.h.in	2011-09-17 15:45:30.000000000 +0200
@@ -38,8 +38,11 @@
 
 /* define if found malloc_stats */
 #undef HAVE_MALLOC_STATS
 
+/* define if found linkat */
+#undef HAVE_LINKAT
+
 /* Define to 1 if you have the <memory.h> header file. */
 #undef HAVE_MEMORY_H
 
 /* readline enabled CLI */
